code_blocks/🔌 Integrations & Events/third-party-integrations/onesignal_2.swift

```diff
@@ -0,0 +1,32 @@
+class AppDelegate: UIResponder, UIApplicationDelegate, OSSubscriptionObserver {
+
+    var window: UIWindow?
+
+    func application(_ application: UIApplication,
+                     didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+
+        Purchases.configure(withAPIKey: "<revenuecat_api_key>", appUserID: nil)
```

Comment:
I think we try to not have this in quotes to force users to add the api key since that would fail to compile otherwise:
```suggestion
        Purchases.configure(withAPIKey: <revenuecat_api_key>, appUserID: nil)
```